lib/modules/manager/npm/update/dependency/pnpm.spec.ts

```diff
@@ -0,0 +1,336 @@
+import { codeBlock } from 'common-tags';
+import * as npmUpdater from '../..';
+
+/**
+ * Per the YAML spec, a document ends with a newline. The 'yaml' library always
+ * uses that when serialising, but `codeBlock` strips the last indentation. This
+ * helper makes assertions simpler.
+ */
+function yamlCodeBlock(
+  literals: TemplateStringsArray,
+  ...placeholders: any[]
+): string {
+  return codeBlock(literals, placeholders) + '\n';
+}
+
+describe('modules/manager/npm/update/dependency/pnpm', () => {
+  it('handles implicit default catalog dependency', () => {
+    const upgrade = {
+      depType: 'pnpm.catalog',
+      depName: 'react',
+      newValue: '19.0.0',
+      managerData: {
+        catalogName: 'default',
+      },
+    };
+    const pnpmWorkspaceYaml = yamlCodeBlock`
+      packages:
+        - pkg-a
+
+      catalog:
+        react: 18.3.1
+    `;
+    const expected = yamlCodeBlock`
+      packages:
+        - pkg-a
+
+      catalog:
+        react: 19.0.0
+    `;
+    const testContent = npmUpdater.updateDependency({
+      fileContent: pnpmWorkspaceYaml,
+      upgrade,
+    });
+    expect(testContent).toEqual(expected);
```

Comment:
```suggestion
    const testContent = npmUpdater.updateDependency({
      fileContent: pnpmWorkspaceYaml,
      upgrade,
    });
    expect(testContent).toEqual(yamlCodeBlock`
      packages:
        - pkg-a

      catalog:
        react: 19.0.0
    `;);
```
Inline expected for better readability. Same goes for the other tests